getuto

```diff
@@ -159,14 +159,19 @@ if [[ ! -d ${GNUPGHOME} ]] ; then
 
 	# TODO: keys.openpgp.org lacks a UID for our keys, need to verify email
 	for keyserver in "${mykeyservers[@]}" ; do
-		gpg ${QUIET_GPG} --batch --keyserver "${keyserver}" --recv-keys ${myrelkeys} || true
+		einfo "Importing release engineering keys from: ${keyserver}"
+		gpg ${QUIET_GPG} --batch --keyserver "${keyserver}" --recv-keys ${myrelkeys} || true &
```

Comment:
Using & means that we inherently discard the exit status -- and `wait` without any options always returns true, not the exit status of the failed command. So it's really a replacement for `|| true`...